5.5/examples/replica/mysql_replica.json

```diff
@@ -158,11 +158,11 @@
                 "env": [
                   {
                     "name": "MYSQL_MASTER_USER",
```

Comment:
I guess i tend to agree that replication_user is clearer, but if we can't change it in the image itself i'm not sure it's worth changing it in the template and then having them out of sync.

perhaps the best thing to do would be to deprecate master_user, and make the image(scripts) support both values?